internal/oauth/oauth.go

```diff
@@ -56,6 +63,13 @@ const (
 	GovClientID = "0oabtyfelbTBdoucy297" // GovClientID for production
 )
 
+func addUserAgentHostParameter() {
+	if !strings.Contains(config.UserAgent, userAgentHostName) {
+		config.UserAgent = fmt.Sprintf("%s;%s)",
```

Comment:
FYI Config values can be overridden by existing env vars:
https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-env-variables/

We do not document user agent but it might make sense to use this mechanism?